### ISSUE TITLE
Allow specifying multiple `ntd_variable` values

### ISSUE THREAD

**Issue by vgXhc** [vgXhc/ntdr#1]
Because the NTD spreadsheet has separate sheets for `UPT`, `VRM` etc., currently the package only reads in one type. Allowing to get, say, `UPT` and `VRM` in a single call could be useful but requires some changes to the structure of the package.

**Comment by vaishaliporthi** [comment]
Does ntd_variable have the following variables - Total operating expenses and Passenger miles/labor hours? If so, what would be the abbreviations for those variables? Please suggest!

**Comment by vgXhc** [comment]
@vaishaliporthi No, these are not available via the package. At the moment the package only pulls data from the two monthly ridership files [adjusted](https://www.transit.dot.gov/ntd/data-product/monthly-module-adjusted-data-release) and [raw](https://www.transit.dot.gov/ntd/data-product/monthly-module-raw-data-release). I'll maybe create a separate issue to update the documentation to make that more obvious.
